prime-router/src/test/kotlin/azure/ApiKeysFunctionsTest.kt

```diff
@@ -95,11 +96,10 @@ class ApiKeysFunctionsTest {
         every { SettingsFacade.common } returns facade
         every {
             facade.putSetting<OrganizationAPI>(
-                any<String>(),
+                organization.name,
```

Comment:
Tightened this mock such that it also confirms the correct parameters are passed.